ray-operator/config/samples/ray_v1alpha1_rayservice-alb-ingress.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: ray-cluster-ingress
+  annotations:
+    alb.ingress.kubernetes.io/scheme: internet-facing
+    alb.ingress.kubernetes.io/tags: Environment=dev,Team=test
+    # See `ingress.md` for more details about how to choose subnets.
+    alb.ingress.kubernetes.io/subnets: subnet-d9d4ff92, subnet-0c0209db843c9fd72
+    alb.ingress.kubernetes.io/target-type: ip
+    # Health Check Settings
```

Comment:
Add more comments explaining why it is necessary to set up health checks and why we have chosen these specific values for the annotations.